### PR TITLE
Add demo video to docs

### PR DESCRIPTION
**Pull request by bastienbeurier**
## Checklist before requesting a review (skip if just adding/editing APIs & templates)
- [ ] I added tests, otherwise the reason is: 
- [ ] I added observability, otherwise the reason is:
- [ ] I added analytics, otherwise the reason is:

### CODE REVIEW

File: docs-v2/introduction.mdx [hunk 1]
```diff
@@ -53,6 +53,10 @@ Nango's flexibility ensures it supports any API integration:
 
 But remember, Nango can work with **any API and any use-case**!
 
+# 📺 Demo video
+
```
Comment (bodinsamuel) [inline docs-v2/introduction.mdx:57]: Nice, would be cool to link the sample app here maybe?
also in the sample app repo replacing the static image by this video

Comment (bodinsamuel) [reply docs-v2/introduction.mdx:57]: not sure why mintlify didn't trigger here

**Review by bodinsamuel — APPROVED** [review]

File: docs-v2/introduction.mdx (hunk 1 above)
Comment (bastienbeurier) [reply docs-v2/introduction.mdx:57]: Agreed, I'll link the video in the sample app. 

About the link to the sample app, are you proposing to add it somewhere in the introduction page? It's already prominently at the top level of the docs, so IMO as discoverable as we can make it, but let me know what you think.

Comment (bodinsamuel) [reply docs-v2/introduction.mdx:57]: yeah true about the menu 👌🏻

**Pull request merged**